dashboard/README.md

```diff
@@ -1,2 +1,141 @@
-# Packit service web UI
+# Packit Service Web UI
 Development moved to separated [Dashboard project](https://github.com/packit-service/dashboard)
+
+
+## UI
+
+### Utils
+
+* SCSS is a superset of CSS with variable support and better query selection. It is compiled to _readable_ CSS.
+
+### UI/UX
+
+Patternfly has two official variants.
```

Comment:
[cockpit project is using patternfly](https://github.com/cockpit-project/cockpit/tree/master/pkg) if you wanna get inspired

Reply:
Yes, that is where I first saw patternfly.